css/style.css

```diff
@@ -0,0 +1,397 @@
+/* layout */
+
+* ::after, ::before {
+    box-sizing: border-box;
+}
+
+.row {
+    display: flex;
+    flex-wrap: wrap;
+}
+
+.col-1 {
+    flex-grow: 1;
+    max-width: 100%;
+}
+
+.col-2 {
+    flex-basis: 16.666667%;
+    max-width: 16.666667%;
+}
+
+.col-3 {
+    flex-basis: 25%;
+    max-width: 25%;
+}
+
+.col-4 {
+    flex-basis: 33.333333%;
+    max-width: 33.333333%;
+}
+
+.col-5 {
+    flex-basis: 41.666667%;
+    max-width: 41.666667%;
+}
+
+.col-6 {
+    flex-basis: 50%;
+    max-width: 50%;
+}
+
+.col-7 {
+    flex-basis: 58.333333%;
+    max-width: 58.333333%;
+}
+
+.col-8 {
+    flex-basis: 66.666667%;
+    max-width: 66.666667%;
+}
+
+.col-9 {
+    flex-basis: 75%;
+    max-width: 75%;
+}
+
+.col-10 {
+    flex-basis: 83.333333%;
+    max-width: 83.333333%;
+}
+
+.col-11 {
+    flex-basis: 91.666667%;
+    max-width: 91.666667%;
+}
+
+.col-12 {
+    flex-basis: 100%;
+    max-width: 100%;
+}
+
+.w-25 {
+    width: 25%;
+}
+
+.w-50 {
+    width: 50%;
+}
+
+.w-75 {
+    width: 75%;
+}
+
+.w-100 {
+    width: 100%;
+}
+
+.flex {
+    display: flex;
+}
+
+.justify-content-end {
+    justify-content: flex-end;
+}
+
+.align-items-end {
+    align-items: flex-end;
+}
+
+.justify-content-between {
+    justify-content: space-between;
+}
+
+.p-1 {
+    padding: 1rem;
+}
+
+.p-2 {
+    padding: 2rem;
+}
+
+.pl-1 {
+    padding-left: 1rem;
+}
+
+.pl-2 {
+    padding-left: 2rem;
+}
+
+.pr-1 {
+    padding-right: 1rem;
+}
+
+.pr-2 {
+    padding-right: 2rem;
+}
+
+.pt-1 {
+    padding-top: 1rem;
+}
+
+.pt-2 {
+    padding-top: 2rem;
+}
+
+.pb-1 {
+    padding-bottom: 1rem;
+}
+
+.pb-2 {
+    padding-bottom: 2rem;
+}
+
+.m-1 {
+    margin: 1rem;
+}
+
+.m-2 {
+    margin: 2rem;
+}
+
+.ml-1 {
+    margin-left: 1rem;
+}
+
+.ml-2 {
+    margin-left: 2rem;
+}
+
+.mr-1 {
+    margin-right: 1rem;
+}
+
+.mr-2 {
+    margin-right: 2rem;
+}
+
+.mt-1 {
+    margin-top: 1rem;
+}
+
+.mt-2 {
+    margin-top: 2rem;
+}
+
+.mb-1 {
+    margin-bottom: 1rem;
+}
+
+.mb-2 {
+    margin-bottom: 2rem;
+}
+
+/* end of layout */
+
+/* html */
+
+html {
+    font-family: sans-serif;
+    -ms-text-size-adjust: 100%; 
+    -webkit-text-size-adjust: 100%; 
+}
+html {
+    display: block;
+}
+
+* {
+    box-sizing: border-box;
+}
+
+/* end of html */
+
+/* head */
+
+head {
+    display:none;
+}
+
+/* end of head */
+
+/* body */
+
+body {
+    margin: 0;
+    display: block;
+}
+
+/* end of body */
+
+/* header */
+
+.header-image {
+    padding: 2rem;
+    margin-bottom: 1rem;
+    background: #3a2929;
+    display: flex;
+    flex-wrap: wrap;
+} 
+
+.w-100 {
```

Comment:
I think the name of this class is misleading, it would make more sense to have called it .w-5. It would also be more logical to place this statement earlier in the file where the other width classes are. This will make it easier to find when you come back to the file in the future.